friends/src/App.js

```diff
@@ -0,0 +1,191 @@
+import React, { Component } from "react";
+import axios from "axios";
+import { Route, Redirect } from "react-router-dom";
+import { FriendsList } from "./components/FriendsList";
+import { AddFriendForm } from "./components/AddFriendForm";
+import { EditFriendForm } from "./components/EditFriendForm";
+import uuid from "uuid";
+import "./App.css";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: [],
+      potentialFriendChanges: {
+        id: "",
+        name: "",
+        age: "",
+        email: "",
+        color: "",
+        favefood: "",
+        quotation: ""
```

Comment:
check out my suggestion from your previous PR: store all these properties and empty string within a object declared outside the component entirely, and anytime you need to reference the empty string version again (resetting the fields, creating a temp reference) you can use that object without having to retype all of this